R/ResampleInstances.R

```diff
@@ -38,3 +38,59 @@ instantiateResampleInstance.RepCVDesc = function(desc, size, task = NULL) {
   g = as.factor(rep(seq_len(desc$reps), each = folds))
   makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds, group = g)
 }
+
+instantiateResampleInstance.FixedCVDesc = function(desc, size, task = NULL) {
+  initial.window = floor(desc$initial.window * size)
+  assertInt(initial.window, lower = 1)
+  if (size - initial.window < desc$horizon)
+    stop(catf("The initial window is %i observations while the data is %i observations. \n
+      There is not enough data left (%i observations) to create a test set for a %i size horizon",
```

Comment:
Since the relative initial window is the only one users touch I am going to leave that as is, but to avoid confusion for us I will change it in the resampleInstance to be `initial.window.abs`